corehq/apps/app_manager/util.py

```diff
@@ -541,12 +541,11 @@ def _app_callout_templates():
 
 
 @quickcache([], timeout=60 * 60 * 24 * 7)  # 1 week
-def _app_callout_templates_ids():
+def app_callout_templates_ids():
     return {t['id'] for t in next(_app_callout_templates())}
 
 
 app_callout_templates = _app_callout_templates()
-app_callout_templates_ids = _app_callout_templates_ids()
```

Comment:
I wonder what the reason was to have this done this way earlier? any thoughts/ideas?

Reply:
I did it that way to be consistent with the way `app_callout_templates` is used, which I presumed was just a nicer way of handling the information (since it's practically static). I'm not aware of any other benefit.

Reply:
But since we're using `quickcache` we get the same benefit (roughly speaking).

Reply:
Okay cool, I thought we were changing something that existed before this PR.